### PR TITLE
Allow traffic from us-central1 private blocks

### PR DESCRIPTION
**Pull request by bskinner**
[DDP-5677](https://broadinstitute.atlassian.net/browse/DDP-5677)

## Context
In order to allow the Salt instances to make use of the ElasticSearch proxy, the allowed IP ranges need to be expanded to include the private address block for `us-central1`. Since all environments are in the `us-central1` data center, these changes will support `dev`, `test` and `prod`.

## Checklist
- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] **N/A** If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] **N/A** If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

## FUD Score
- [x] :relaxed: All good, business as usual!

## How do we demo these changes?
- [x] This ticket adds no user-visible features.

## Testing
- [x] I have written zero automated tests but have poked around locally to verify proper functionality

## Release
- [x] These changes require no special release procedures--just code!

### CODE REVIEW

**Review by yufengwng — DISMISSED** [review]

**Comment by bskinner** [comment]
@yufengwng Can I get another approval on this? It also looks like there's no workflow defined in CircleCI for these changes, so the checks will never complete. Unless there are objections, I'm going to bypass the checks and merge anyway after this is approved.

**Comment by yufengwng** [comment]
You can run the tests using the script in `ddp-study-server/pepper-apis/scripts`:
```
$ ./scripts/run_ci_tests.sh <your_branch>
```

**Review by yufengwng — APPROVED** [review]

**Pull request merged**
